mypy/semanal.py

```diff
@@ -3327,7 +3327,10 @@ def name_not_defined(self, name: str, ctx: Context) -> None:
     def name_already_defined(self, name: str, ctx: Context,
                              original_ctx: Optional[SymbolTableNode] = None) -> None:
         if original_ctx:
-            extra_msg = ' on line {}'.format(original_ctx.node.get_line())
+            if original_ctx.node and original_ctx.node.get_line() != -1:
```

Comment:
IIUC the idea, then I would check for ``original_ctx.mod_id``, and if it is not ``None`` then mention the module where original definition occurred. Otherwise, I would just set ``extra_msg = ''``.